wireguard/wireguard.go

```diff
@@ -345,3 +351,21 @@ func apply(n *config.Node, c *wgtypes.Config) error {
 
 	return wg.ConfigureDevice(ncutils.GetInterfaceName(), *c)
 }
+
+// GetRealIface - used only for darwin OS retrieves tun iface based on reference iface name from config file
+func GetRealIface(iface string) (string, error) {
```

Comment:
Is this duplicating functionality in localport.go  getRealInterface?  maybe that func should be exported and used instead of creating a new function.

Reply:
unused func removing it